dotnet/samples/README.md

```diff
@@ -0,0 +1,10 @@
+## Kernel Samples
+
+| Type                                                                        | Description                                                                                              |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| [`Getting started`](./GettingStarted/README.md)                             | A single step-by-step tutorial to get started                                                            |
```

Comment:
Capitalization here: `[`Getting Started`](./GettingStarted/README.md)`

How about: `Take this step by step tutorial to get started with the Semantic Kernel and get introduced to the key concepts.`